runner/progress.py

```diff
@@ -52,21 +60,39 @@ def update_stats(self):
         if len(self._test_info) == 0:
             return
 
-        total_score = total_std = 0
-        for scores in self._scores.values():
-            score, std = mean_std(scores)
-            total_score += score
-            total_std += std
-        self._label.config(text=f"{next(self._blinker)} Score: {total_score:.1f} ± {total_std:.1f}")
+        total_tokens = [info["span"] for info in self._test_info.values()]
+        total_tokens += [self._memory_span] * (self._num_tests - len(total_tokens))
+        total_tokens = sum(total_tokens)
 
+        # Progress bar shows the overall progress
         if self._isolated:
             progress = len(self._test_info) / self._num_tests
         else:
-            total = [info["span"] for info in self._test_info.values()]
-            total += [self._memory_span] * (self._num_tests - len(total))
             progress = sum(min(max(0, self._at - info["start"]), info["span"]) for info in self._test_info.values())
-            progress /= max(sum(total), 1)
+            progress /= max(total_tokens, 1)
         self._progressbar["value"] = int(100 * progress)
+
+        # First row shows the total score so far and the tests completed
+        num_finished = 0
+        total_score = 0
+        for scores in self._scores.values():
+            for s in scores:
+                total_score += s / self._num_reps
```

Comment:
If I am reading this right, `num_reps` shouldn't be universal. Sure it is a part of the config, but that number can be overridden at the test level (again in the config). So while there may be nominally 3 repetitions of a test, I could have it so that there is only one example of ChapterBreak for example

Reply:
That's absolutely right. I think the same, but I didn't know how to get around it... oh, maybe I have. Give me a minute 😬 🤞🏾